platooning_control/src/platoon_control.cpp

```diff
@@ -99,6 +103,22 @@ namespace platoon_control
 
     bool PlatoonControlPlugin::controlTimerCb()
     {
+        // If it has been a long time since input data has arrived then reset the input counter and return
+        // Note: this quiets the controller after its input stream stops, which is necessary to allow 
+        // the replacement controller to publish on the same output topic after this one is done.
+        long current_time = ros::Time::now().toNsec() / 1000000;
+        if (current_time - prev_input_time_ > config_.shutdownTimeout)
```

Comment:
I suggest we add some log statements to see what is value of "current_time - prev_input_time_" so we can tune the parameter

Reply:
Good idea.